app/utils/auth.py

```diff
@@ -21,4 +21,5 @@ def key_check(api_key: str = Header(None)):
     set_key = "dev"
   
   if api_key is None or api_key != set_key:
+    print("@@@@@API KEY DIFFERENT")
```

Comment:
This is not how you are supposed to debug. Please use logger and use proper error messages.